pkg/agent/controller/agent.go

```diff
@@ -89,9 +99,43 @@ func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) runtime.Object
 			},
 		},
 	}
+	err = a.updateExportedServiceStatus(svcExport)
```

Comment:
Of course the problem with updating it here is that it hasn't actually been synced yet and could fail - I guess this is where we need https://github.com/submariner-io/admiral/issues/66 😄  I guess we can move this call once that is done.

Reply:
yes that is right, we can change this one then.